apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -71,6 +71,15 @@ public function register_meta_field() {
 		register_meta( 'post', '_starter_page_template', $args );
 	}
 
+	/**
+	 * Register rest api endpoint for side-loading images.
+	 */
+	public function register_rest_api() {
+		require_once __DIR__ . '/class-wp-rest-sideload-image-controller.php';
+
+		( new \WP_REST_Sideload_Image_Controller() )->register_routes();
```

Comment:
what do you think about adding a [class_exist](https://www.php.net/manual/en/function.class-exists.php) checking?

```php
if ( class_exists( 'WP_REST_Sideload_Image_Controller' ) {
  // ...
}
```

Reply:
I don't think we need a `class_exists` check, we're requiring the file just one line above.

@getdave Rather than defining it in the global namespace, let's add the namespace declaration in `class-wp-rest-sideload-image-controller.php'`, since everything this plugin does should be namespaced

Reply:
> we're requiring the file just one line above.

(Playing devil's advocate...) We've just seen how this can go wrong...it wouldn't hurt to have the extra check?

Reply:
That's why we're fixing it :) I much rather it throws an error and tells us that something is wrong, than it failing silently and the functionality just disappears.